src/tests/Microsoft.PowerFx.Interpreter.Tests/DatabaseSimulationTests.cs

```diff
@@ -142,6 +142,12 @@ public override void Visit(IValueVisitor visitor)
             {
                 throw new NotImplementedException("TestEntityValue.Visit");
             }
+
+            public override string ToExpression()
+            {
+                // !JYL! Do we need to serialize this?
```

Comment:
>/ !JYL! Do we need to serialize this? [](http://example.com/codeflow?start=17&length=37)

No need for this one

Reply:
this is in testing. 
The good news is this is actuall an example for custom serialization :) 
Ie, when we serialize a DataverseRecordValue, we'll want to override and serialize as `Lookup(table, id=Guid(id))` 
. That's an ideal compact form, rather than serialize as a full record.